### PR TITLE
Add post on the train stations of Paris

### PR DESCRIPTION
**Pull request by emilyreese**
This post explains Paris's cool train station culture, with one train station per region of France/neighboring countries. :bullettrain_side: :euro: 

cc @erakor

### CODE REVIEW

File: _posts/2015-11-05-Les-Gares-de-Paris.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Les Gares de Paris
+
+Il y a six gares principales à Paris : Gare de Lyon, Gare du Nord, Gare de l'Est, Gare Saint Lazare, Gare d'Austerlitz, et Gare Montparnasse. Les gares sont toutes jolies, et chaque gare a sa région ! Pratique, non ? D'ailleurs elles sont toutes mieux que Penn Station. Il est utile de se familiariser avec ces gares.
```
Comment (erakor) [inline _posts/2015-11-05-Les-Gares-de-Paris.md]: "D'ailleurs, "

Maybe mentioning where is Penn Station. Not everyone knows NYC 😉

File: _posts/2015-11-05-Les-Gares-de-Paris.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+### Rive droit:
```
Comment (erakor) [inline _posts/2015-11-05-Les-Gares-de-Paris.md]: Rive _droite_ (rive est féminin)

File: _posts/2015-11-05-Les-Gares-de-Paris.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+*Gare de Lyon*: cette gare se trouve dans le 12ème arrondissement. Elle est reconnue par sa belle tour de l'horloge. La plupart des trains à Gare de Lyon sont à déstination du Sud-Est de la France, genre autour de Lyon (voilà le nom de la gare) ! Les lignes 1 et 14 du métro et A et D du RER viennent aussi à Gare de Lyon.
```
Comment (erakor) [inline _posts/2015-11-05-Les-Gares-de-Paris.md]: destination

File: _posts/2015-11-05-Les-Gares-de-Paris.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+*Gare du Nord*: Gare du Nord est immense, avec le plus de voyageurs par an en Europe. Ses trains et lignes desservent la Belgique, l'Angleterre...des pays du nord quoi ! Il est également possible de rattraper les lignes 4, 5, et 2 (à La Chapelle) du métro et les lignes B, D, et E du RER à cette gare.
+
+*Gare de l'Est*: à deux pas de Gare du Nord, Gare de L'Est est la gare principle pour l'est de la France et l'Allemagne. Elle était originalement appelée Gare de Strasbourg. Les lignes 4, 5, et 7 du métro et la ligne E du RER sont disponibles à Gare de l'Est.
```
Comment (erakor) [inline _posts/2015-11-05-Les-Gares-de-Paris.md]: _originalement_ is not very the best word here. I'd say 

> Elle était à l'origine appelée

File: _posts/2015-11-05-Les-Gares-de-Paris.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+*Gare Saint Lazare*: Cette gare est située dans le 8ème arrondissement de Paris. Elle dessert la Normandie principalement mais est très important aussi pour le métro : les lignes 3, 9, 12, 13, et 14 sont toutes trouvables à Gare Saint Lazare (la ligne E du RER aussi).
```
Comment (erakor) [inline _posts/2015-11-05-Les-Gares-de-Paris.md]: > importante (bc _la_ gare 😛)

**Comment by emilyreese** [comment]
@erakor :bow: I made some edits with your suggestions. `initialement` instead of `originalement`, ça te va?

**Comment by erakor** [comment]
lgtm
Let's change the name to get the date of today.
Your text is so good!

**Comment by emilyreese** [comment]
@erakor file has been renamed!

**Pull request merged**